integration/airflow/tests/test_openlineage_adapter.py

```diff
@@ -128,11 +129,12 @@ def test_emit_start_event():
                 namespace="default",
                 name="job",
                 facets={
-                    "documentation": DocumentationJobFacet(description="description"),
-                    "jobType": JobTypeJobFacet(processingType="BATCH", integration="AIRFLOW", jobType="TASK"),
+                    "documentation": documentation_job.DocumentationJobFacet(description="description"),
+                    "jobType": job_type_job.JobTypeJobFacet(
+                        processingType="BATCH", integration="AIRFLOW", jobType="TASK"
+                    ),
                 },
             ),
-            producer=_PRODUCER,
```

Comment:
If we're removing this producer line from tests, i would remove it also from the adapter so it's not passed to RunEvent. Right now the producer is still passed explicitly, so it can stay in the tests as well.

Reply:
I removed it from the adapter.